internal/util/logging/logram.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// LogRAM implements zap logging entry interception and stores the last 1024 entries in the ring buffer in memory.
+package logging
+
+import (
+	"fmt"
+	"sync"
+
+	"go.uber.org/zap/zapcore"
+)
+
+var LogRAM *logRAM
+
+// logRAM is a storage of log records in memory.
+type logRAM struct {
+	mu    sync.RWMutex
+	log   []*zapcore.Entry
+	index int64
+}
+
+// NewLogRAM creates entries log in memory.
```

Comment:
Whould it be possible to document what does size mean? E.g. is it in bytes?

Reply:
We don't know size of the buffer, it depends on size of the messages. 
Only quantity 1024

Reply:
Yeah, you are right. Then, the comment should be something like "size specifies the capacity (number of messages) of the buffer". I think I misread the task.